src/transport_performance/urban_centres/raster_uc.py

```diff
@@ -44,7 +53,7 @@ class UrbanCentre:
     ------
     TypeError
         When `file` is not either of string or pathlib.Path.
-    FileExistsError
+    FileNotFoundError
```

Comment:
Will change this in `defence.py` as I copied it from its docstrings.

Reply:
I think this has been handled in a different issue and merged to dev. But that happened when you were away on hols, will need to check this PR is up to speed with changes in dev.

Reply:
Hi @r-leyshon, I created this branch this week from what I thought was the up-to-date dev branch, so not sure why it didn't include these changes. Please let me know if you find something out as I might be doing something wrong in Git!